about.html

```diff
@@ -0,0 +1,112 @@
+                <img class="logo" src="images/logo.jpg" alt="logo">
+            </a>
+            <div class="d-m-container">
+                <a class="desktop-m tab-0" href="about.html">About</a>
+                <a class="desktop-m tab-1" href="#">Program</a>
+                <a class="desktop-m tab-2" href="#">Join</a>
+                <a class="desktop-m tab-3" href="#">Sponsor</a>
+                <a class="desktop-m tab-4" href="#">News</a>
+                <a class="desktop-m tab-5" href="#">CC Campaign</a>
+            </div>
+        </nav>
+    </header>
+    <header class="mobile-only">
+        <a><i class="fas fa-bars hamburger" id="menu-harmbuger"></i></a>
+    </header>
+    <main>
+        <section class="header-intro">
+            <p class="p-1">When the world comes together, we create a better tomorrow</p>
+            <h1 class="h-1">Dubai Expo 2021</h1>
+            <h3 class="h-3">Be part of the magic</h3>
+
+            <p class="p-2">The Expo 2020 TV gives you a glimpse into the exciting experiences you can enjoy at the World’s Greatest Show and is dedicated to our programmes, partners, attractions, pavilions, events, news and much more.</p>
+            <p class="p-3-d">Please contact us per email or via calls for any further questions <br> about Dubai expo, 2021 <br> gabrielcoder247@gmail.com</p>
+
+        </section>
+        <section class="activity">
+            <h2 class="h-1-act">Dubia Creative expo 2021, <br> Global common summit</h2>
+            <p class="p-1-act">The expo was decide due to the pandemic to take place from 1st-30th, Oct, 2021</p>
+            <div class="expo-1">
+                <img src="images/expo-2.png" alt="expo">
+                <p class="p-img">starting from, 1st -30th, Oct, 2021 | Dubai</p>
+            </div>
+            <hr>
+
+        </section>
+        <section class="participants">
+            <h2 class="h-1-part">See the past participants <br> in Dubai expos</h2>
+            <p class="p-1-part">take a look at these two Global Expos that took place in Italy and Russia</p>
+            <div class="past-event">
+                <div class="event e-card-1">
+                    <img src="images/2020.jpg" alt="2020">
+                    <h2 class="year">2019</h2>
+                    <p class="e-desc">Global expo in Italy 2019</p>
+                </div>
+                <div class="event e-card-2">
+                    <img src="images/2019.jpg" alt="2019">
+                    <h2 class="year">2020</h2>
+                    <p class="e-desc">Global expo in Russia 2020</p>
+                </div>
+
+            </div>
+
+        </section>
+        <section class="partner">
+
+        </section>
+        <footer>
+            <div class="contact">
+                <p class="contact-a">Dubai Expo 2021</p>
+                <p class="contact-b">Dubai expo <br> Some right reserved </p>
+
+            </div>
+        </footer>
```

Comment:
- [ ] On mobile view, the footer has a white background but on your own, it is not changing on mobile view. Please fix that.